eras/shelley-ma/impl/src/Cardano/Ledger/ShelleyMA/Core.hs

```diff
@@ -0,0 +1,24 @@
+{-# LANGUAGE FlexibleContexts #-}
+
+module Cardano.Ledger.ShelleyMA.Core
+  ( ShelleyMAEraTxBody (..),
+    module Cardano.Ledger.Shelley.Core,
+  )
+where
+
+import Cardano.Ledger.Core (Era (..), EraTxBody (..), Value)
+import Cardano.Ledger.Mary.Value (MultiAsset (..))
+import Cardano.Ledger.Shelley.Core (ShelleyEraTxBody)
```

Comment:
So, here we no longer need to import `Cardano.Ledger.Core`, but we re-export the whole of `Cardano.Ledger.Shelley.Core`:
```suggestion
```
```suggestion
import Cardano.Ledger.Mary.Value (MultiAsset (..))
import Cardano.Ledger.Shelley.Core
```